2.cellprofiler_analysis/README.md

```diff
@@ -16,3 +16,6 @@ cd 2.cellprofiler_analysis
 source nf1_analysis.sh
 ```
 
+To run analysis on plates 1 and 2, it took approximately one hour. 
+To run analysis on plates 3 and 3 prime, it took approximately 31 hours.
```

Comment:
Double checking: is `prime` here defined anywhere or "common-enough" to be understood by those in the audience for this project? If not, consider defining it somewhere. Please feel free to ignore.

Reply:
It isn't really defined, but I will add to the main README the name 3 prime so that it known that is just what the plate is called.